src/functions.php

```diff
@@ -387,24 +387,6 @@ function is_last_pipeline_operator_write(array $pipeline): bool
     return $key === '$merge' || $key === '$out';
 }
 
-/**
- * Return whether the "out" option for a mapReduce operation is "inline".
- *
- * This is used to determine if a mapReduce command requires a primary.
- *
- * @internal
- * @see https://mongodb.com/docs/manual/reference/command/mapReduce/#output-inline
- * @param string|array|object $out Output specification
- */
-function is_mapreduce_output_inline(string|array|object $out): bool
-{
-    if (! is_array($out) && ! is_object($out)) {
-        return false;
-    }
-
-    return array_key_first(document_to_array($out)) === 'inline';
-}
```

Comment:
I'll miss you, little utility function.

![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExdzNrdjJ4ZWptcnYxOTEyMnpybHRoNXF6cnZ2czVhMnBreGtkN2ZlcyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/UQaRUOLveyjNC/giphy.gif)